integration/integration-end-to-end.spec.ts

```diff
@@ -103,9 +103,12 @@ test("Upload configuration file, choose form, fill form, submit form correctly",
   // Check that download exists
   await t.expect(Title.textContent).contains("Document(s) issued successfully");
   await t.expect(Selector("div").withText("Document-1.tt").exists).ok();
+  // await t.expect(Selector("a[download='Document-1.tt']").exists).ok();
```

Comment:
eh? why commented out ah?

Reply:
we not using a tag for download liao, forgot to remove it, will remove it now.